test/task_helper_functions.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+find_files() {
+  local pth="$1"
+  shift
+    find "${pth}" '(' \
+    -path '*/.git' -o \
+    -path '*/.terraform' -o \
+    -path '*/.kitchen' -o \
+    -path '*.svg' -o \
+    -path '*.png' -o \
+    -path './autogen' -o \
+    -path './test/fixtures/all_examples' -o \
```

Comment:
Why are we excluding these? Not clear why this repo needs a customized find_files.